content/master/getting-started/provider-alibabacloud-part-2.md

```diff
@@ -0,0 +1,670 @@
+---
+title: Alibaba Cloud Quickstart Part 2
+weight: 120
+tocHidden: true
+aliases:
+  - /master/getting-started/provider-alibabacloud-part-2
+---
+
+{{< hint "important" >}}
+This guide is part 2 of a series.
+
+[**Part 1**]({{<ref "provider-alibabacloud" >}}) covers
+installing Crossplane and connecting your Kubernetes cluster to Alibaba Cloud.
+
+{{< /hint >}}
+
+This guide walks you through building and accessing a custom API with Crossplane.
+
+## Prerequisites
+
+* Complete [quickstart part 1]({{<ref "provider-alibabacloud" >}}) connecting Kubernetes
+  to Alibaba Cloud.
+* an Alibaba Cloud account with permissions to create an ECS instance, VPC, VSwitch and SecurityGroup.
+
+{{<expand "Skip part 1 and just get started" >}}
```

Comment:
looks like there is an error on this line causing the build/deployment to the preview site to fail:
```
11:33:27 AM: Error: error building site: assemble: "/opt/build/repo/content/master/getting-started/provider-alibabacloud-part-2.md:25:1": failed to extract shortcode: shortcode "expand" must be closed or self-closed
```

This section of the contributing guide should help: https://docs.crossplane.io/contribute/features/#hide-long-outputs